### ISSUE TITLE
Does COCO dataset need psa and seg step?

### ISSUE THREAD

**Issue by vtars** [xulianuwa/MCTformer#25]
Does COCO dataset need psa and seg step? 
I haven't seen the COCO dataset code in ./psa and ./seg, and I got a poor result on COCO dataset (about 8% on initial seed).
How to reproduce the 42.0 mIoU in paper on COCO dataset?
Thanks!

**Comment by xulianuwa** [comment]
> Does COCO dataset need psa and seg step? I haven't seen the COCO dataset code in ./psa and ./seg, and I got a poor result on COCO dataset (about 8% on initial seed). How to reproduce the 42.0 mIoU in paper on COCO dataset? Thanks!

Yes. Will add the rest steps for COCO soon. The initial seed shouldn't have that low mIoU. What's the classification mAP you got on the validation set?

**Comment by Bala93** [comment]
Hi everyone, Isn't the miou reported in paper the validation data's miou post deeplab training ?

**Comment by Italy2006** [comment]
> Hi, on th COCO14, I achieve 36.x% seed mIoU, however, I can not achieve considerable performances in terms of mask mIoU with IRN post-processing, can you please provide more code or param details for COCO setting. Thanks. btw, what is the mask mIoU of your MCTformer+PSA in coco14 train set？

Hi,have you reproduced his results? I want to know some specif details of the settings, thank you!

**Comment by Italy2006** [comment]
> > Does COCO dataset need psa and seg step? I haven't seen the COCO dataset code in ./psa and ./seg, and I got a poor result on COCO dataset (about 8% on initial seed). How to reproduce the 42.0 mIoU in paper on COCO dataset? Thanks!
> 
> Yes. Will add the rest steps for COCO soon. The initial seed shouldn't have that low mIoU. What's the classification mAP you got on the validation set?

I'm sorry, but could you provide the setting of seg step about traing on COCO dataset? Thank you

**Comment by fu-roc** [comment]
> Hi, on th COCO14, I achieve 36.x% seed mIoU, however, I can not achieve considerable performances in terms of mask mIoU with IRN post-processing, can you please provide more code or param details for COCO setting. Thanks. btw, what is the mask mIoU of your MCTformer+PSA in coco14 train set？

Have you solved this problem? I encountered the same problem when reproducing this paper.
